contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -201,7 +201,7 @@ contract JBETHPaymentTerminalStore {
     @dev
     Mint's the project's tokens according to values provided by a configured data source. If no data source is configured, mints tokens proportional to the amount of the contribution.
 
-    @dev 
```

Comment:
It may be worth making this formatting change separately against main, getting it merged, and then rebasing your changes on top of it to make this diff easier.

Reply:
I like this idea

https://github.com/jbx-protocol/juice-contracts-v2/pull/44